controllers/yawol-controller/loadbalancerset/loadbalancerset_controller.go

```diff
@@ -169,6 +176,39 @@ func (r *LoadBalancerSetReconciler) reconcileStatus(
 		return ctrl.Result{RequeueAfter: time.Second * 2}, nil
 	}
 
+	// Write set contains master condition
```

Comment:
Can we combine the status updates into a single one?

Reply:
Do you mean combining this with the two above status updates?
The above two `if`s always return